File: src/Test/RecordsFFICheatcode.js
export const runRecordsFFICheatcode = function(limit) {
  let n = Math.floor(limit);
  let obj = { a: 1, b: { c: 2, d: 3 } };
  for (let i = 0; i < n; i++) {
    obj.b.c += 1;
  }
  return obj.b.c - 2;
};

File: src/Test/RecordsFFICheatcode.rs
pub fn Test_RecordsFFICheatcode_runRecordsFFICheatcode(mut limit: i64) -> i64 {
    let mut a = 0;
    let mut c = 0;
    let mut e = 0;
    let mut f = 0;
    while limit > 0 {
        a += 1;
        c += 2;
        e += 3;
        f += limit % 5;
        limit -= 1;
    }
    f
}
